MANIFEST.in

```diff
@@ -16,8 +16,7 @@ recursive-include tests *.py
 exclude vectors
 recursive-exclude vectors *
 
-exclude azure-pipelines.yml .azure-pipelines .travis.yml .travis
-recursive-exclude .azure-pipelines *
+exclude .travis.yml .travis
```

Comment:
Probably need to add `.github` here?

Reply:
Done, although this probably means we've been packaging .github for a few releases 😆